backend/battle/tests/test_view.py

```diff
@@ -74,3 +80,343 @@ def test_verify_if_not_returns_winner_missing_pokemons_teams(self):
 
         with self.assertRaises(IndexError):
             get_winner_for(self.team_creator, self.team_opponent)
+
+
+class ListBattlesTest(TestCaseUtils):
+    def setUp(self):
+        super().setUp()
+        self.battle = baker.make('battle.Battle', creator=self.user)
+        self.battle_from_user = Battle.objects.filter(creator=self.user)
+
+    def test_login_user_can_acess_battle_list(self):
+        response = self.auth_client.get(reverse('battle_list'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_view_returns_empty_list(self):
+        battle = Battle.objects.get(creator=self.user)
+        battle.delete()
+        response = self.auth_client.get(reverse('battle_list'))
+        response_qs = response.context_data.get('battle_list')
+        self.assertCountEqual(response_qs, [])
+
+    def test_view_returns_one_battle_in_list(self):
+        response = self.auth_client.get(reverse('battle_list'))
+        response_qs = response.context_data.get('battle_list')
+        self.assertCountEqual(response_qs, [self.battle_from_user][0])
```

Comment:
Remove the `self.battle_from_user` by the actual query

Reply:
Great!